fe/fe-core/src/main/java/org/apache/doris/statistics/Statistics.java

```diff
@@ -225,12 +199,40 @@ public void mockTableStatsWithRowCount(long tableId, long rowCount) {
         }
 
         if (tableStats.getRowCount() != rowCount) {
-            tableStats.setRowCount(rowCount);
+            // tableStats.setRowCount(rowCount);
         }
     }
 
-    // Used for unit test
-    public void putTableStats(long id, TableStats tableStats) {
-        this.idToTableStats.put(id, tableStats);
+    /**
+     * if the table stats is not exist, create a new one.
+     *
+     * @param tableId table id
+     * @return @TableStats
+     */
+    private TableStats getNotNullTableStats(long tableId) {
```

Comment:
Do we have machanism to remove the stats of deleted table currently?

Reply:
Not yet, it is planned to be implemented with automatic update of statistics in the future.

Reply:
fine